ark/utils/spatial_lda_utils_test.py

```diff
@@ -0,0 +1,79 @@
+import os
+import numpy as np
+import pandas as pd
+import tempfile
+import pytest
+from ark.utils.spatial_lda_utils import *
+
+
+def test_check_format_cell_table_args():
+    # Testing variables
+    base_cols = [
+        "point",
+        "label",
+        "cell_size",
+        "centroid-0",
+        "centroid-1",
+        "pixelfreq_hclust_cap",
+        "name",
+        "Au",
+        "CD4",
+        "CD8"
+    ]
+    # Cell table pd.DataFrame
+    VALID_DF = pd.DataFrame(columns=base_cols)
+    # Not a pd.DataFrame
+    INVALID_DF1 = []
+    # Doesn't meet minimum column requirements
+    INVALID_DF2 = pd.DataFrame(columns=base_cols[1:6])
+
+    # Markers
+    VALID_MARKERS = ["Au", "CD4", "CD8"]
+    # Specifies marker not included in cell table
+    INVALID_MARKERS1 = ["Au", "CD4", "CD8", "Vimentin"]
+    # Includes integer
+    INVALID_MARKERS2 = ["Au", "CD4", "CD8", 3]
+    # Empty list
+    INVALID_MARKERS3 = []
+
+    # Cluster
+    VALID_CLUSTERS = [1, 2, 3]
+    # Strings instead of integers
+    INVALID_CLUSTERS1 = ["a", "b", "c"]
+    # Empty List
+    INVALID_CLUSTERS2 = []
+
+    # FOVs
+    VALID_FOVS = [1, 2, 3]
+    # Strings instead of integers
+    INVALID_FOVS1 = ["a", "b", "c"]
+    # Empty List
+    INVALID_FOVS2 = []
+
+    # Run Checks
+    with pytest.raises(ValueError):
```

Comment:
I believe these tests will pass as long as at least one of these raises a ValueError; it won't require that they each do. You should double check this to make sure I'm not wrong though